scout/load/phenotype.py

```diff
@@ -19,7 +20,7 @@
 LOG = logging.getLogger(__name__)
 
 
-def load_hpo(
+def load_phenotypes(
```

Comment:
Renamed, because otherwise it's confusing. this function loads BOTH HPO and OMIM terms

Reply:
Right, this is where it really convoluted, since we rely on HPO to provide us connection to diseases (OMIM/ORPHANET - maybe POSSUM?) and use both for the same command.  This one is really the setup command. We should just move that step into `scout setup` (maybe even have the setup do a download everything) and have it call the separated `scout.load.phenotype.load_hpo_terms` and `scout.load.disease.load_disease_terms` from there.

Reply:
Let's do this in another PR, I feel one is already messy as it is..

Reply:
https://github.com/Clinical-Genomics/scout/issues/4128